### DISCUSSION TITLE
Logs of LSP communication

### DISCUSSION THREAD

**Discussion by RudolfVonKrugstein** [helix-editor/helix#7203] [Q&A] [answered]
Is there a way of getting the logs from the communication between helix and the LSP?
That would be very useful, I am often struggling with LSPs and when using neovim these logs often helped very myc.

**Comment by pascalkuthe** [comment, marked as answer]
Running helix in verbose mode outputs the LSP logs to the editor log

**Comment by niknbr** [comment]
Couldn't find the documentation anywhere, so for anyone facing this, start helix as `hx -v` and run command `:log-open`

**Reply by johannesrave** [reply]
another "trick" that is helping me with live-lsp-debugging is
```
tail --follow ~/.cache/helix/helix.log
```
this will "watch" the logfile in a terminal window, and you can get direct feedback eg. for LSP actions.
